statify.php

```diff
@@ -20,6 +20,7 @@
 define( 'STATIFY_FILE', __FILE__ );
 define( 'STATIFY_DIR', dirname( __FILE__ ) );
 define( 'STATIFY_BASE', plugin_basename( __FILE__ ) );
+define( 'STATIFY_VERSION', '1.6.3' );
```

Comment:
Shouldn't this be 1.7.0?

Reply:
Absolutely (if we don‘t add this to 1.6.x - and I don‘t think so - or postpone this to 1.8 obviously)